### PR TITLE
Feat/build per entrypoint

### PR DESCRIPTION
**Pull request by iamstarkov**

### CODE REVIEW

**Comment by iamstarkov** [comment]
@dimchez i might need your help with ["require.ensure without ignoreChunks" test case](https://github.com/nordnet/nordnet-release-plugin/blob/master/test/index.test.js#L103-L116)

File: src/index.js [hunk 1]
```diff
@@ -49,10 +49,9 @@ function done(stats) {
...
   }
 
-  const content = Object.keys(chunks)
+  Object.keys(chunks)
     .filter(key => !contains(ignore, key))
-    .map(key => inject(key))
-    .join('');
-
-  fs.writeFileSync(`${dir}/base.js`, content);
+    .map(key => {
```
Comment (dimchez) [inline src/index.js]: Technically, that's a `forEach`, not a `map`. But works either way, just clearer intentions IMHO.

Comment (iamstarkov) [inline src/index.js]: u r 💯 right

Comment (iamstarkov) [inline src/index.js]: fixed

**Pull request merged**